examples/index.html

```diff
@@ -1,39 +1,9 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset='utf-8'>
-    <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <link href='https://fonts.googleapis.com/css?family=Chivo:900' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="/stylesheets/stylesheet.css" media="screen" />
-    <link rel="stylesheet" type="text/css" href="/stylesheets/pygment_trac.css" media="screen" />
-    <link rel="stylesheet" type="text/css" href="/stylesheets/print.css" media="print" />
-    <!--[if lt IE 9]>
-    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
-    <title>JSON Lines Examples</title>
-  </head>
+---
+layout: default
+title: Examples
+---
```

Comment:
If we rename it, then we have to rewrite everything in markdown :) it's a bit more work and I can't guarantee it will be exactly the same display as the current one, but I can try something close. is that good?

Reply:
Thanks! I will have a look